src/app/modules/backbone/backbone.common.ts

```diff
@@ -0,0 +1,27 @@
+import { Routes } from "@angular/router";
+
+import { AppComponent } from "@app/app.component";
+import { BackbonePageComponent } from "@modules/backbone/pages/backbone/backbone.page";
+import { BackboneComponent } from "@modules/backbone/components/backbone/backbone.component";
+
+export const pageDeclarations: any[] = [
+    BackbonePageComponent,
+];
+
+export const componentDeclarations: any[] = [
+    BackboneComponent
+];
+
+export const providerDeclarations: any[] = [
+];
+
+export const routes: Routes = [
+    {
+        path: "",
+        component: BackbonePageComponent,
+        children: [
+            { path: "", component: AppComponent },
+            { path: "backbone", component: BackboneComponent },
```

Comment:
both of these routes are not needed: the appcomponent will not contain any html, but will rather load only a router-outlet. It is bootstrapped automatically on app load so there is no need to add it here.

The children route here will contain routes to components that will apread "around" the sidebar and the footer. They will components like product-list, product-detail etc.